packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -82,17 +82,22 @@ const propTypes = {
    * The function returning the frame html reference.
    */
   refCallback: PropTypes.func,
+  /**
+   *  To determine if menu is opened inside the popup
+   */
+  isMenu: PropTypes.bool,
 };
 
 const defaultProps = {
   classNameInner: null,
   contentHeightMax: -1,
   contentWidthMax: -1,
   isFocusedDisabled: false,
-  isHeaderDisabled: false,
+  isHeaderDisabled: true,
```

Comment:
Default value should be false only. Since prop name is isHeaderDisabled it should remove the header when true but as per your implementation it will add header when true

Reply:
Changed it to false